### PR TITLE
Feat/collapsable list

### PR DESCRIPTION
**Pull request by Thienhuynh95**
## Jira issue

[Create List Component - Amino](https://myzonos.atlassian.net/browse/LCG-802)

## Description

This PR converts VStack, HStack and List to functional component and adds new Collapsable List component

## Todo

- [x] Bump version and add tag

### CODE REVIEW

**Review by goodpickle — CHANGES_REQUESTED** [review]
What do you think about adding s a CheckIcon for the selected list item? I remember seeing that in the designs, but I didn't see that in the stories.

File: src/components/list/CollapsableList.tsx [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+      padding: var(--amino-space-quarter);
+      border: 1px solid var(--amino-gray-l60);
+      border-radius: var(--amino-radius-xl);
+    `}
+
+  ${({ withNegativeMargin }) =>
+    withNegativeMargin &&
+    css`
+      margin: var(--amino-space-negative);
+    `} /** Collapsable List item */
```
[the remark this answers is not in the document]
Comment (Thienhuynh95) [reply src/components/list/CollapsableList.tsx:48]: Yes, that was my bad. I'm just seeing it this morning, I'll push the change, thanks for catching that 😉

[the remark this answers is not in the document]
Comment (Thienhuynh95) [reply src/components/list/CollapsableList.tsx]: About your comment above. Please correct me if I'm wrong, I think the CheckIcon you see for the selected list item is Rich Radio/RichCheckbox. But for the list or collapsable list, I don't see the CheckIcon for selected item except for the background color.
image

[the remark this answers is not in the document]
Comment (Thienhuynh95) [reply src/components/list/CollapsableList.tsx]: Here is the design of the list component, I assume the selected state for the list item of the collapsable list would be the same 🤔 
https://www.figma.com/file/WnKnmG7L3Q74hqPsw4rbEE/Amino-2.0?node-id=349%3A7128

[the remark this answers is not in the document]
Comment (goodpickle) [reply src/components/list/CollapsableList.tsx]: Thanks for pulling that up. Must have been a design relic or my mistake. 👍

**Review by goodpickle — DISMISSED** [review]

**Review by goodpickle — APPROVED** [review]

**Pull request merged**